.github/workflows/pipeline.yml

```diff
@@ -46,6 +48,28 @@ jobs:
         version: v1.0.0
     - name: Test GoReleaser
       run: make test-release
+  test-e2e:
+    strategy:
+      matrix:
+        k8s-version: ["kindest/node:v1.21.2", "kindest/node:v1.20.7"]
+        os: [ubuntu-latest]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Create k8s Kind Cluster
+      uses: helm/kind-action@v1.2.0
+      with:
+        cluster_name: kind
+        node_image: ${{ matrix.k8s-version }}
+    - name: Install operator-sdk
+      run: |
+        mkdir -p "$GITHUB_WORKSPACE/bin"
+        curl -o "$GITHUB_WORKSPACE/bin/operator-sdk" -L https://github.com/operator-framework/operator-sdk/releases/download/v0.17.1/operator-sdk-v0.17.1-x86_64-linux-gnu
```

Comment:
Need to change v0.17.1 to v0.18.2. Its breaking/error out (Line No: 68)

Reply:
Fixed in commit 7a1f6f2875b0aa31a81258cfb16ffa76ce587b2a